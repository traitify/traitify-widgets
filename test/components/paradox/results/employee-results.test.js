import Component from "components/paradox/results/employee-results";
import ComponentHandler from "support/component-handler";

jest.mock("components/results/personality/archetype/heading", () => (() => (<div className="mock">Personality Archetype Heading</div>)));
jest.mock("components/results/personality/archetype/skills", () => (() => (<div className="mock">Personality Archetype Skills</div>)));
jest.mock("components/results/personality/archetype/tips", () => (() => (<div className="mock">Personality Archetype Tips</div>)));
jest.mock("components/results/personality/dimension/list", () => (() => (<div className="mock">Personality Dimension List</div>)));
jest.mock("lib/with-traitify", () => ((value) => value));

describe("Paradox.EmployeeResults", () => {
  it("renders component", () => {
    const component = new ComponentHandler(<Component />);

    expect(component.tree).toMatchSnapshot();
  });
});
