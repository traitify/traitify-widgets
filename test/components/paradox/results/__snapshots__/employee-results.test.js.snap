// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paradox.EmployeeResults renders component 1`] = `
<section
  className="container"
>
  <div
    className="mock"
  >
    Personality Archetype Heading
  </div>
  <div
    className="mock"
  >
    Personality Archetype Skills
  </div>
  <div
    className="mock"
  >
    Personality Archetype Tips
  </div>
  <div
    className="mock"
  >
    Personality Dimension List
  </div>
</section>
`;
